docs-src/README.md.template

```diff
@@ -0,0 +1,19 @@
+# ![logo](https://quintilesims.github.io/layer0/static/logo_rs.png)
```

Comment:
this should be removed -> can use `sed` for this instance.

Reply:
👍